src/Console/InstallCommand.php

```diff
@@ -67,6 +57,10 @@ public function handle()
         }
 
         $this->info('Dusk scaffolding installed successfully.');
+
+        $this->info('Installing ChromeDriver binary...');
+
+        $this->call('dusk:chrome-driver');
```

Comment:
We should use the `--all` option here. If you install Laravel & Dusk on a Windows/Mac machine and then run the tests in Homestead, you aren't using the latest ChromeDriver.

Reply:
Hah, I was doubting to do that but you're right. I've updated this.